specification/eventgrid/data-plane/readme.md

```diff
@@ -111,118 +111,35 @@ openapi-type: data-plane
 tag: package-2018-01
 ```
 
-### Tag: package-2018-01
+### Tag: package-2024-06-01
 
-These settings apply only when `--tag=package-2018-01` is specified on the command line.
+These settings apply only when `--tag=package-2024-06-01` is specified on the command line.
 
-``` yaml $(tag) == 'package-2018-01'
+``` yaml $(tag) == 'package-2024-06-01'
 input-file:
-- Microsoft.Storage/stable/2018-01-01/Storage.json
```

Comment:
Reformatting order of api - versions from recent to least recent and removing system events from namespace versions

Reply:
will be done in separate pr